spec/index.html

```diff
@@ -3531,30 +3531,39 @@ <h2>A Canonical form of N-Quads</h2>
       <code>predicate</code>, 
       <code>object</code>,
       and <code>graphLabel</code>,
-      each of which MUST be a single space (<code>U+0020</code>).</li>
+      each of which MUST be a single space (code point <code class="codepoint">U+0020</code>).</li>
     <li><a data-cite="RDF11-CONCEPTS#dfn-literal">Literals</a> with the
       datatype <code>http://www.w3.org/2001/XMLSchema#string</code>
       MUST NOT use the datatype IRI part of the <a data-cite="N-QUADS#grammar-production-literal">literal</a>,
       and are represented using only <a data-cite="N-QUADS#grammar-production-STRING_LITERAL_QUOTE">STRING_LITERAL_QUOTE</a>.
     </li>
-    <li><code><a data-cite="N-QUADS#grammar-production-HEX">HEX</a></code> MUST use only uppercase letters (<code>[A-F]</code>).</li>
-    <li>Within <a data-cite="N-QUADS#grammar-production-STRING_LITERAL_QUOTE">STRING_LITERAL_QUOTE</a>,
-      the characters
-      <code>U+0008</code> (<code title="BACKSPACE"><sub>BS</sub></code>),
-      <code>U+0009</code> (<code title="HORIZONTAL TAB"><sub>HT</sub></code>),
-      <code>U+000A</code> (<code title="LINE FEED"><sub>LF</sub></code>),
-      <code>U+000C</code> (<code title="FORM FEED"><sub>FF</sub></code>),
-      <code>U+000D</code>  (<code title="CARRIAGE RETURN"><sub>CR</sub></code>),
-      <code>U+0022</code> (<code title="DOUBLE QUOTE">&quot;</code>), and
-      <code>U+005C</code> (<code title="BACKSLASH">\</code>)
-      MUST be encoded using <code><a data-cite="N-QUADS#grammar-production-ECHAR">ECHAR</a></code>.
-      Characters in the range from <code>U+0000</code> to <code>U+001F</code>
-      and <code>U+007F</code>  (<code title="delete"><sub>DEL</sub></code>)
-      that are not represented using <code><a data-cite="N-QUADS#grammar-production-ECHAR">ECHAR</a></code>
-      MUST be represented by <code><a data-cite="N-QUADS#grammar-production-UCHAR">UCHAR</a></code>.
-      All other characters MUST be represented by their native [[UNICODE]] representation.</li>
-    <li>The token <code><a data-cite="N-QUADS#grammar-production-EOL">EOL</a></code> MUST be a single <code>U+000A</code>.</li>
-    <li>The final <code><a data-cite="N-QUADS#grammar-production-EOL">EOL</a></code> MUST be provided.</li>
+    <li><code><a data-cite="N-QUADS#grammar-production-HEX">HEX</a></code> MUST use only digits `[0-9]` and uppercase letters (<code>[A-F]</code>).</li>
```

Comment:
```suggestion
    <li><code><a data-cite="N-QUADS#grammar-production-HEX">HEX</a></code> MUST use only digits (<code>[0-9]</code>) and uppercase letters (<code>[A-F]</code>).</li>
```
to align with the other parts like the subsequent `(<code>[A-F]</code>)`